gems/activerecord/6.0/activerecord.rbs

```diff
@@ -825,7 +825,8 @@ module ActiveRecord
       #
       #   person.pets.size  # => 5 # size of the collection
       #   person.pets.count # => 0 # count from database
-      def build: (?_EachPair attributes) ?{ () -> untyped } -> untyped
+      def build: (?Array[_EachPair] attributes) ?{ () -> untyped } -> Array[untyped]
+               | (?_EachPair attributes) ?{ () -> untyped } -> untyped
```

Comment:
It looks like you have created the possibility to return an Array even when there are no arguments, is this intentional?

```rb
person.pets.build #=> Array[untyped]
```

Reply:
Good point. It's my mistake. Fixed now.

Reply:
Finally I appended the Array case to the tail. It's better to return a single entry if `untyped` value given.